### PR TITLE
Added OWNERS file

### PR DESCRIPTION
**Pull request by pecameron**
First draft to get this started.

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@rajatchopra @dcbw PTAL

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - JacobTanenbaum
+  - pecameron
+  - squeed
+  - dcbw
+  - rajatchopra
+  - shettyg
```
[the remark this answers is not in the document]
Comment (pecameron) [reply OWNERS]: Done

**Comment by pecameron** [comment]
@rajatchopra PTAL

**Comment by pecameron** [comment]
@dcbw @rajatchopra @shettyg is there more to be done here? Is this ready to merge?

File: OWNERS (hunk 1 above)
[the remark this answers is not in the document]
Comment (aserdean) [reply OWNERS]: can you add @alinbalutoiu as well?

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  - aserdean
+approvers:
```
Comment (aserdean) [inline OWNERS]: me and @russellb also have merge rights.

Comment (pecameron) [reply OWNERS]: Added approvers @aserdean @russellb

Comment (pecameron) [reply OWNERS]: @aserdean Added reviewer: alinbalutoiu

**Pull request merged**